llvm/lib/CodeGen/CodeGenTargetMachineImpl.cpp

```diff
@@ -163,6 +166,10 @@ CodeGenTargetMachineImpl::createMCStreamer(raw_pwrite_stream &Out,
         getTargetTriple(),
         Options.MCOptions.OutputAsmVariant.value_or(MAI.getAssemblerDialect()),
         MAI, MII, MRI);
+    for (StringRef Opt : Options.MCOptions.InstPrinterOptions)
+      if (!InstPrinter->applyTargetSpecificCLOption(Opt))
+        return createStringError(inconvertibleErrorCode(),
+                                 "invalid InstPrinter option '" + Opt + "'");
```

Comment:
---
(nit) There is an overload accepting `Twine`:
```suggestion
        return createStringError("invalid InstPrinter option '" + Opt + "'");
```